### DISCUSSION TITLE
Is there a way for two sampler to use same seed?

### DISCUSSION THREAD

**Discussion by tzekeong** [comfyanonymous/ComfyUI#83] [Q&A]
Is there a way for two sampler to use same seed?

**Comment by GrimblyGorn** [comment]
Try the file I posted [here](https://github.com/comfyanonymous/ComfyUI/discussions/128#discussioncomment-5369863). It adds 2 new samplers with externalized seed connections and a seed node to use with them. Simply rename that file. to .py again and drop it in the "custom_nodes" folder then restart and it should work. All the new nodes will be in the "sampling" and new "sampling/vars" menus.

To be fair, they aren't new samplers so much as modded samplers from the original nodes.py file that comes with comfyUI. So credits to everyone else here, and WAS since this is modded from his originally. They should solve your issue regardless of their origin though :)

**Comment by fdupas67** [comment]
I have found a way to use the same seed in Multiple Ksampler. You can create a node Seed everywhere which will allow you to use the same seed in every Ksampler so you are sure to use always the same seed.
